giskard_vision/core/dataloaders/tfds.py

```diff
@@ -0,0 +1,159 @@
+from typing import Any, Dict, List, Optional
+
+import numpy as np
+
+from giskard_vision.core.dataloaders.base import DataIteratorBase
+from giskard_vision.core.dataloaders.meta import MetaData
+from giskard_vision.landmark_detection.types import Types
+from giskard_vision.utils.errors import GiskardImportError
+
+
+def flatten_dict_exclude_wrapper(
+    d: Dict[str, Any], excludes: List[str] = [], parent_key: str = "", sep: str = "_", flat_np_array: bool = False
+) -> Dict[str, Any]:
+    """
+    Flattens a nested dictionary without the specified keys.
+
+    Args:
+        d (Dict[str, Any]): The dictionary to flatten.
+        excludes (List[str]): Keys to exclude from the flattened dictionary.
+        parent_key (str): The base key string for the flattened keys.
+        sep (str): The separator between keys.
+
+    Returns:
+        Dict[str, Any]: The flattened dictionary.
+    """
+    items = {}
+    for k, v in d.items():
+        if k in excludes:
+            continue
+        items.update(flatten_dict({k: v}, parent_key=parent_key, sep=sep, flat_np_array=flat_np_array).items())
+
+    return items
```

Comment:
I tried to exclude it before because there could be too many fields with flattened np array.

Let me try to exclude them directly in `flatten_dict` thus.